src/lexer/src_string.sig

```diff
@@ -1,7 +1,9 @@
 signature SRC_STRING =
 sig
     type yypos = int
-
+    exception StringNotClosed of int * int (* line position * line number *)
```

Comment:
I take it back. That said, as a general strategy we are saving only the total character offsets and using the `Newline` structure to map them to line positions.